modules/oceanbase/src/main/java/org/testcontainers/oceanbase/OceanBaseCEContainer.java

```diff
@@ -43,6 +84,28 @@ public OceanBaseCEContainer(DockerImageName dockerImageName) {
         dockerImageName.assertCompatibleWith(DEFAULT_IMAGE_NAME);
 
         addExposedPorts(SQL_PORT, RPC_PORT);
+        setWaitStrategy(Wait.forLogMessage(".*boot success!.*", 1));
+    }
+
+    @Override
+    protected void configure() {
+        addEnv("MODE", mode.name().toLowerCase());
+
+        if (!DEFAULT_USER_TENANT_NAME.equals(tenantName)) {
+            if (mode == Mode.SLIM) {
+                throw new IllegalArgumentException("Tenant name is not configurable on slim mode");
```

Comment:
What happen if this is provided? will the container fail? If that's not the case WDYT about just logging a warning about the tenant name being ignored?

Reply:
Updated.